helm/harvester/charts/harvester/templates/deployment.yaml

```diff
@@ -81,9 +82,9 @@ spec:
               chmod +x /data/harvester/run-harvester-crons;
               cp /opt/harvester/sandbox/condor_config.local /data/harvester/;
               {{- if .Values.noRoot }}
-              /opt/harvester/etc/rc.d/init.d/run-harvester-services;
-              {{- else }}
               runuser -u atlpan -g zp -- /opt/harvester/etc/rc.d/init.d/run-harvester-services;
+              {{- else }}
```

Comment:
Is this logic correct? noRoot means that the container is launched with non-root pid so that it cannot use runuser.

Reply:
I thought that noRoot means the script run-harvester-service should not run as root. Ok I will update it.

Reply:
Because of misunderstanding of noRoot, I updated several places. I will update them.